File: crates/accuracy/fixtures/typescript/variables.ts
// Const bindings and arithmetic.

const a = 1;
const b = a + 2; //~ depends: a@3
const c = a + b; //~ depends: a@3, b@4

function main(): number {
    const d = c; //~ depends: c@5
    return d + 1; //~ depends: d@8
}

File: crates/accuracy/fixtures/rust/variables.rs
// Let bindings, arithmetic and shadowing.

fn main() {
    let a = 1;
    let b = a + 2; //~ depends: a@4
    let c = a + b; //~ depends: a@4, b@5
    let d = c; //~ depends: c@6
    let a = d + 1; //~ depends: d@7
    let e = a; //~ depends: a@8
    let _ = e; //~ depends: e@9
}
